.github/workflows/mac.yml

```diff
@@ -88,7 +88,7 @@ jobs:
     uses: ./.github/workflows/self-hosted-runner-select.yml
     secrets: inherit
     with:
-      github-hosted-runner-label: macos-13
+      github-hosted-runner-label: macos-15
```

Comment:
I think this will generate arm instead of x86 binaries. Can we not lower SDK requirements by patching/hardcoding different lower version in SM?

Reply:
Yes, that's my plan.